### PR TITLE
macOS: fixes app termination

### LINKED ISSUE #1117: macOS: Closing window doesn't happen until the mouse cursor has moved
Version: 0.20.0-alpha3
OS: Mojave 10.14.6
Code: window example

Clicking on the red button in tile bar doesn't close the window until mouse cursor has moved.

Video: http://88.99.85.67:1337/bug264.mp4

### PR DESCRIPTION
**Pull request by vbogaevsky**
Resolves #1117 
- [x] Added an entry to CHANGELOG.md if knowledge of this change could be valuable to users
- [x] Tested on all platforms changed
- [x] `cargo fmt` has been run on this branch

### CODE REVIEW

**Comment by vbogaevsky** [comment]
@Osspial ok, I won't disturb her again.
Should I add `Alpha 5` or is it ok to put the changes to `Alpha 4`?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Unreleased
+- On macOS, fix application termination on `ControlFlow::Exit`
```
Comment (Osspial) [inline CHANGELOG.md]: Could you add a space between this line and `Unreleased` so this keeps consistent formatting?

**Comment by vbogaevsky** [comment]
Checked it with multi-window example. After pressing red button, one window closes instantly, but the other one hangs for several events. Clearly, I've missed something.

**Comment by vbogaevsky** [comment]
@Osspial do I merge this PR?

**Comment by Osspial** [comment]
Sure, go ahead.

**Pull request merged**